.gitignore

```diff
@@ -41,6 +41,7 @@ _UpgradeReport_Files/
 ipch/
 *.sdf
 *.opensdf
+node.VC.opendb
```

Comment:
Sure, I can remove this, but I believe this is an artifact of vs 2015, so anyone using this IDE will need to add this to their .gitignore.  Is there precedent here?